### PR TITLE
M kovalsky/reportrebindallmultipleworkspaces

### PR DESCRIPTION
**Pull request by m-kovalsky**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,21 @@
-# semantic-link-labs
+# [semantic-link-labs](https://semantic-link-labs.readthedocs.io/en/0.5.0/)
```
Comment (eisber) [inline README.md:1]: are you doing breaking changes? why the change to 0.5?

File: src/sempy_labs/_connections.py [hunk 2]
```diff
@@ -60,24 +61,24 @@ def create_connection_cloud(
...
-            "Connection Path": o["connectionDetails"]["path"],
-            "Privacy Level": o["privacyLevel"],
-            "Credential Type": o["credentialDetails"]["credentialType"],
-            "Single Sign On Type": o["credentialDetails"]["singleSignOnType"],
-            "Connection Encryption": o["credentialDetails"]["connectionEncryption"],
-            "Skip Test Connection": o["credentialDetails"]["skipTestConnection"],
+            "Connection Id": o.get("id"),
+            "Connection Name": o.get("name"),
+            "Connectivity Type": o.get("connectivityType"),
+            "Connection Type": o.get("connectionDetails").get("type"),
```
Comment (eisber) [inline src/sempy_labs/_connections.py:67]: that change seems duplicate to the other PR. let's wait for either of the one to merged into main.

**Review by eisber — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (m-kovalsky) [reply README.md:1]: there are no breaking changes but i've added quite a bit of new functions.
